build/filters/Uno.UI-packages-skia.slnf

```diff
@@ -6,7 +6,6 @@
       "AddIns\\Uno.UI.MSAL\\Uno.UI.MSAL.Skia.csproj",
       "AddIns\\Uno.UI.MediaPlayer.Skia.Gtk\\Uno.UI.MediaPlayer.Skia.Gtk.csproj",
       "AddIns\\Uno.UI.Svg\\Uno.UI.Svg.Skia.csproj",
-      "AddIns\\Uno.WinUI.Graphics3DGL\\Uno.WinUI.Graphics3DGL.csproj",
```

Comment:
Why is this entry removed?

Reply:
We are producing the package in the job "Building WinAppSDK Graphics3DGL package binaries" (it will really be building both for WinAppSDK and Skia actually). There is no need to build it in the Skia jobs.

Reply:
I'm waiting for CI to make sure the generated package is correct.

Reply:
Makes sense thanks!